python/ray/autoscaler/aws/cloudwatch/ray_prometheus_waiter.sh

```diff
@@ -0,0 +1,21 @@
+MAX_ATTEMPTS=120
```

Comment:
Two things about this file --

1. 

I think it needs a shebang line to make the linter happy.

2.

Instead of rsync-ing the script, you can include it in the Ray installation by it adding to [setup.py](https://github.com/ray-project/ray/blob/2e99fb215fb3b064cbbff47dd38f52e1f3bdda55/python/setup.py#L165)